devops/tasks.py

```diff
@@ -159,14 +159,13 @@ def release(
         component = Component(path)
         if path in images:
             component.image = images[path]
-            del images[path]
+            images.pop(path, None)
```

Comment:
Since the above `if` requires this to be in `images` I might remove the `None` from here as it should never be needed and might mask some weird bugs in the future?

Same with the `if path in tags`

Reply:
true